acceptance/tests/language/pcore_resource_types_should_have_precedence_over_ruby.rb

```diff
@@ -1,4 +1,9 @@
 test_name 'C98097 - generated pcore resource types should be loaded instead of ruby for custom types' do
+
+tag 'audit:medium',
+    'audit:integration',
```

Comment:
Yes! I have added the tag, but the github UI is not displaying it in the comment summary. Double check the diff.